src/main/java/com/jcabi/w3c/DefaultCssValidator.java

```diff
@@ -64,6 +68,8 @@
         this.uri = entry.toString();
     }
 
+    // @todo introduces a test case to validade both process and sucess
```

Comment:
@jrdalpra rultor will reject it 
use `@todo #20:30min`
also the puzzle should contains at least 20 words